app/views/products/index.html.erb

```diff
@@ -2,14 +2,16 @@
 
 <span>
     <% @products.each do |product| %>
-        <img src= <%= product.photo_url %> alt="image of #{product.name}">
-        <ul>
-            <li><h3><%= link_to product.name, product_path(product.id) %></h3></li>
-            <% if false %>
-            currently commented out.  need help with pathway to user/product_id
-            <li><%= link_to product.user_id, user_path(user.id) %></li> %>
-            <% end %>
-            <li><%= link_to product.description, product_path(product.id) %></li>
-        </ul>
+        <% if product.retired == false %>
+            <img src= <%= product.photo_url %> alt="image of #{product.name}">
+            <ul>
+                <li><h3><%= link_to product.name, product_path(product.id) %></h3></li>
+                <% if false %>
+                currently commented out.  need help with pathway to user/product_id
```

Comment:
I think we had decided that we were going to link all products to just /product/:id (product show) instead of nesting the links (as per the example we were going off). Definitely willing to open it up for discussion, though! 

(Also realized I was doing single comments instead of starting a review #sorry for the emails)

Reply:
Hm...I think Mina worked on this method. I think the product_path(product.id) is the correct path. I'm not sure where that comment came from haha

Reply:
Okay, I'll follow up with Mina on this! It looks like we can probably just delete the line but I'll double-check with her.